src/components/ChangePlan/Checkout/Discounts/Discounts.test.js

```diff
@@ -0,0 +1,209 @@
+import { Discounts } from './Discounts';
+import { fakeAccountPlanDiscounts } from '../../../../services/doppler-account-plans-api-client.double';
+import { render, cleanup, waitFor, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import IntlProvider from '../../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+
+const mockedhandleChange = jest.fn();
+
+const DiscountsElement = ({ discountsList, sessionPlan, selectedPlanDiscount, disabled }) => (
+  <IntlProvider>
+    <Discounts
+      discountsList={discountsList}
+      sessionPlan={sessionPlan}
+      selectedPlanDiscount={selectedPlanDiscount}
+      disabled={disabled}
+      handleChange={mockedhandleChange}
+    />
+  </IntlProvider>
+);
+
+describe('Discount component', () => {
+  afterEach(cleanup);
+
+  it('should show discount message when subscription is quarterly or higher', async () => {
+    // Arrange
+    const fakeSessionPlan = {
+      idPlan: 42,
+      planSubscription: 6,
+    };
+
+    // Act
+    const { container } = render(
```

Comment:
Why `container` ? there is another easier way . [See Guide Lines for more information](https://github.com/FromDoppler/doppler-webapp#test)

Reply:
I used ```container``` because I needed find an element by class.

Reply:
for example: ```container.querySelector('.dp-wrap-subscription')```. Do you know another way to find it?